bears/css/CSSAutoPrefixBear.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CSSAutoPrefixBear(Lint, LocalBear):
+    executable = "postcss"
+    arguments = "--use autoprefixer {filename}"
```

Comment:
We never test if autoprefixer is installed - which means the test will fail instead of being skipped